### PR TITLE
Update Test Verification section

### PR DESCRIPTION
**Pull request by dpatil-magento**
## Description

TODO: Describe your changes in detail here.

## Related Issue

Closes #ISSUE_NUMBER.

### Verification Steps

1. Go to the FOO page.
2. Verify the BAR shows up.
3. Make sure BAZ does a thing.

## Checklist

-   I have added tests to cover my changes, if necessary.
-   I have added translations for new strings, if necessary.
-   I have updated the documentation accordingly, if necessary.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+-   Verify user is able to sort data after applying filters on category page
+-   Verify user is able to sort data after applying filters on search page
+
+#### Test scenario(s) for any Magento Backend Supported Configurations -
+Example
+- Update default Sort value in backend and repeat above scenarios
+
+#### Does Browser/Device testing needed?
+Example
+- Yes, Browser testing is needed as X UI component is impacted
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:65]:
```suggestion
- Yes, browser testing is needed as X UI component may be impacted on <browser>
- Yes, device testing is needed as X UI component may be impacted on <mobile|desktop|etc>
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+#### Test scenario(s) for any existing impacted features/areas - 
+Example
+-   Verify user is able to sort data after applying filters on category page
+-   Verify user is able to sort data after applying filters on search page
+
+#### Test scenario(s) for any Magento Backend Supported Configurations -
+Example
+- Update default Sort value in backend and repeat above scenarios
+
+#### Does Browser/Device testing needed?
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:63]:
```suggestion
#### Is Browser/Device testing needed?
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 3]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+Example
+-   Verify user is able to apply filters on category page
+-   Verify user is able to apply filters on search page
+
+#### Test scenario(s) for any existing impacted features/areas - 
+Example
+-   Verify user is able to sort data after applying filters on category page
+-   Verify user is able to sort data after applying filters on search page
+
+#### Test scenario(s) for any Magento Backend Supported Configurations -
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:59]:
```suggestion
#### Test scenario(s) for any Magento Backend Supported Configurations
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+<!-- So, we request you to add detailed test plan on what needs to be QA'ed before this PR gets merged -->
+<!-- As you add the test plan you would also come across a scenario which might needs to checked -->
+<!-- Also, as part of Review/QA we will update test plan if needed -->
+
+#### Test scenario(s) for direct fix/feature - 
+Example
+-   Verify user is able to apply filters on category page
+-   Verify user is able to apply filters on search page
+
+#### Test scenario(s) for any existing impacted features/areas -
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:54]:
```suggestion
#### Test scenario(s) for any existing impacted features/areas
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
-2. Verify the BAR shows up.
-3. Make sure BAZ does a thing.
+## Test Plan
+
+<!-- We look forward to have no/less bugs part of our code review and QA -->
+<!-- So, we request you to add detailed test plan on what needs to be QA'ed before this PR gets merged -->
+<!-- As you add the test plan you would also come across a scenario which might needs to checked -->
+<!-- Also, as part of Review/QA we will update test plan if needed -->
+
+#### Test scenario(s) for direct fix/feature -
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:49]:
```suggestion
#### Test scenario(s) for direct fix/feature
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+
+#### Test scenario(s) for any Magento Backend Supported Configurations -
+Example
+- Update default Sort value in backend and repeat above scenarios
+
+#### Does Browser/Device testing needed?
+Example
+- Yes, Browser testing is needed as X UI component is impacted 
+
+#### Any adhoc/edge case scenarios needs to be considered?
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:67]:
```suggestion
#### Any ad-hoc/edge case scenarios that need to be considered?
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
-
-<!-- Please describe in detail how a reviewer can verify your changes, -->
-<!-- OR how you will demonstrate the changes to the stakeholder(s). -->
-
-1. Go to the FOO page.
-2. Verify the BAR shows up.
-3. Make sure BAZ does a thing.
+## Test Plan
+
+<!-- We look forward to have no/less bugs part of our code review and QA -->
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:44]:
```suggestion
<!-- During code review and QA we will try to ensure there are no bugs introduced by this change -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+<!-- So, we request you to add detailed test plan on what needs to be QA'ed before this PR gets merged -->
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:45]:
```suggestion
<!-- So, we request that you add a detailed test plan on what needs to be checked before this PR gets merged -->
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -39,14 +39,34 @@ Closes #ISSUE_NUMBER.
...
+<!-- As you add the test plan you would also come across a scenario which might needs to checked -->
+<!-- Also, as part of Review/QA we will update test plan if needed -->
```
Comment (sirugh) [inline .github/PULL_REQUEST_TEMPLATE.md:47]:
```suggestion
<!-- Feel free to update this after submitting the PR as you discover new scenarios -->
<!-- As part of review/QA we may also add or update the test plan if necessary-->
```

**Review by sirugh — DISMISSED** [review]
I like it. Some suggestions :)

**Review by anthoula — APPROVED** [review]

**Pull request merged**